src/main/java/planes/ExperimentalPlane.java

```diff
@@ -0,0 +1,48 @@
+package planes;
+
+import models.ClassificationLevel;
+import models.ExperimentalType;
+
+import java.util.Objects;
+
+public class ExperimentalPlane extends Plane {
+
+    private final ExperimentalType type;
+    private ClassificationLevel classificationLevel;
+
+    public ExperimentalPlane(String model, int maxSpeed, int maxFlightDistance, int maxLoadCapacity, ExperimentalType type, ClassificationLevel classificationLevel) {
```

Comment:
Please reformat this part of the code, a lot of arguments is hard to read on a single line

Reply:
done